src/Compilers/CSharp/Portable/Lowering/Instrumentation/DebugInfoInjector_SequencePoints.cs

```diff
@@ -207,7 +196,7 @@ private static BoundExpression AddConditionSequencePoint(BoundExpression conditi
             // Add hidden sequence point unless the condition is a constant expression.
             // Constant expression must stay a const to not invalidate results of control flow analysis.
             var valueExpression = (condition.ConstantValueOpt == null) ?
-                new BoundSequencePointExpression(syntax: null, expression: factory.Local(local), type: condition.Type) :
+                new BoundSequencePointExpression(syntax: null!, expression: factory.Local(local), type: condition.Type) :
```

Comment:
Why is it ok to pass null here?

Reply:
Sequence points are allowed to have null syntax - this is checked by assertion in the BoundNode.

Don't like it much, but not wanting to change that at this point.